Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:8.16-alpine as builder
+
+# Install SSL ca certificates
+#RUN apk update && apk add git && apk add ca-certificates
+
+# Create appuser
+#RUN adduser -D -g '' appuser && mkdir /go-analyzer
+RUN mkdir /javascript-analyzer
+
+# source code
+WORKDIR /javascript-analyzer
+
+# get the rest of the source code
+COPY . /javascript-analyzer
```

Comment:
This can be simplified to the `COPY . .`. You are already in the `javascript-analyzer` directory, because of the previous `WORKDIR` command.